### PR TITLE
MG - migrate to play 2.5 (continuation of #70 by mchv)

### PR DESCRIPTION
**Pull request by mario-galic**
Note this PR is to be merged into #70 by @mchv. 

It seems to work in CODE.

There are two FIXMEs:

- Are `gu.management` metrics necessary? Isn't cloudwatch sufficient? This PR disables them - see `httpFilters` definition.
- `GoogleDoc.getCsvForDoc` seems to be dead code.

### CODE REVIEW

File: app/controllers/Application.scala [hunk 1]
```diff
@@ -1,17 +1,15 @@
 package controllers
 
+import play.api.Configuration
 import play.api.mvc._
-import play.api.Play
-
-class Application extends Controller {
+import play.api.routing.Router
 
+class Application(configuration: Configuration, router: => Router) extends Controller {
```
Comment (twrichards) [inline app/controllers/Application.scala]: I would perhaps add a comment about the call-by-name `=>` as it's non-obvious but took us a while to figure out

**Comment by mchv** [comment]
@mario-galic any reason why you close that PR and not merge your changes to my original `PR`?

**Comment by mario-galic** [comment]
@mchv I was unsure about merging into someone else's PR without the explicit +1 from the author of the target PR.

**Comment by mchv** [comment]
@mario-galic you have my explicit +1 to do so! Feel free to rebase, cherry-pick commits or close my PR and open a new one if easier.

**Pull request closed without merging**